resources/data/qgis-hackfests.json

```diff
@@ -2,26 +2,26 @@
+{ "type": "Feature", "properties": { "year": 2012, "month": "April", "hackfest_number": "7", "place": "Lyon", "notes": "Tim stayed up till 4am hacking then nearly missed his flight!", "month_int": 4, "day_int": 1, "date_nice": "2012\/04\/01" }, "geometry": { "type": "Point", "coordinates": [ 4.831924438476562, 45.758062221258413 ] } },
+{ "type": "Feature", "properties": { "year": 2012, "month": "October", "hackfest_number": "8", "place": "Essen", "notes": "First hackfest in Essen, first hackfest for Matthias, Geometry v2 was introduced.", "month_int": 10, "day_int": 1, "date_nice": "2012\/10\/01" }, "geometry": { "type": "Point", "coordinates": [ 7.112553030272008, 51.430481213249188 ] } },
+{ "type": "Feature", "properties": { "year": 2013, "month": "April", "hackfest_number": "9", "place": "Valmeira", "notes": "Lots of socialising on the lovely walk to and from the hackfest venue each day", "month_int": 4, "day_int": 1, "date_nice": "2013\/04\/01" }, "geometry": { "type": "Point", "coordinates": [ 25.386449569327571, 57.543518905433167 ] } },
+{ "type": "Feature", "properties": { "year": 2013, "month": "September", "hackfest_number": "10", "place": "Brighton", "notes": "New QGIS website theme was introduced. We launched QGIS 2.0 at FOSS4G after the hackfest! Get mapping get mapping!", "month_int": 9, "day_int": 1, "date_nice": "2013\/09\/01" }, "geometry": { "type": "Point", "coordinates": [ -0.088517781844579, 50.866844644193002 ] } },
+{ "type": "Feature", "properties": { "year": 2014, "month": "April", "hackfest_number": "11", "place": "Wien", "notes": "Combined OSGEO and QGIS Hackfest. 140GB of data used.", "month_int": 4, "day_int": 1, "date_nice": "2014\/04\/01" }, "geometry": { "type": "Point", "coordinates": [ 16.370315551757812, 48.248454577306291 ] } },
+{ "type": "Feature", "properties": { "year": 2014, "month": "October", "hackfest_number": "12", "place": "Essen", "notes": "Second Linux Hotel hackfest! We introduced plans for QGIS LTR Releases.", "month_int": 10, "day_int": 1, "date_nice": "2014\/10\/01" }, "geometry": { "type": "Point", "coordinates": [ 7.112553030272008, 51.430481213249188 ] } },
+{ "type": "Feature", "properties": { "year": 2015, "month": "May", "hackfest_number": "13", "place": "Nødebo", "notes": "First QGIS User Conference. We had supper in a tipi! Forest parkour was the outside activity of choice.", "month_int": 5, "day_int": 1, "date_nice": "2015\/05\/01" }, "geometry": { "type": "Point", "coordinates": [ 12.345521450042723, 55.989587335710141 ] } },
+{ "type": "Feature", "properties": { "year": 2015, "month": "November", "hackfest_number": "14", "place": "Las Palmas", "notes": "Matthias and Marco were surfing at lunch break everyday....and not writing code during that time.", "month_int": 11, "day_int": 1, "date_nice": "2015\/11\/01" }, "geometry": { "type": "Point", "coordinates": [ -15.423848072810554, 28.110973325973482 ] } },
```

Comment:
Heyyyy!! there is the information missing that we always compensated for that by staying late in the evenings!